website/src/content/api-reference/commercetools-frontend-application-shell-connectors.mdx

```diff
@@ -0,0 +1,319 @@
+---
+title: "@/application-shell-connectors"
+---
+
+<Subtitle>
+
+Components to work with the `@commercetools-frontend/application-shell` package to access and subscribe to specific data within the AppShell.
+
+</Subtitle>
+
+# Installation
+
+```
+yarn add @commercetools-frontend/application-shell-connectors
+
+# or
+
+npm --save install @commercetools-frontend/application-shell-connectors
+```
+
+Additionally install the peer dependencies (if not present)
```

Comment:
I guess keeping these up-to-date is hard-ish. Maybe we can have a link to a `package.json`. I presume many users would just read this and 🤔.

Reply:
Eventually these should be automated like we do in uikit. It's a bit trickier doing it here with the Gatsby setup but totally doable.